auth-server/build.gradle

```diff
@@ -0,0 +1,21 @@
+version = '0.0.1-SNAPSHOT'
```

Comment:
Product file names should follow the mask: <artefact_name>-<build_number>.extention

It looks like we should use '0.0.1.SNAPSHOT', @susau is it correct?